test/contrib/sqla_test.py

```diff
@@ -252,10 +257,66 @@ class ModSQLATask(sqla.CopyToTable):
             connection_string = CONNECTION_STRING
             table = "item_property"
             column_separator = ","
+            chunk_size = 1
 
             def requires(self):
                 return ModBaseTask()
 
         task1, task2 = ModBaseTask(), ModSQLATask()
-        luigi.build([task1, task2], local_scheduler=True)
+        luigi.build([task1, task2], local_scheduler=True, n_workers=self.NUM_WORKERS)
+        self._check_entries(self.engine)
+
+    def test_update_rows_test(self):
+        class ModBaseTask(luigi.Task):
+
+            def output(self):
+                return MockFile("BaseTask", mirror_on_stderr=True)
+
+            def run(self):
+                out = self.output().open("w")
+                for task in TASK_LIST:
+                    out.write("dummy_" + task)
+                    #out.write(task)
+                out.close()
+
+        class ModSQLATask(sqla.CopyToTable):
+            connection_string = CONNECTION_STRING
+            table = "item_property"
+            columns = [
+                (["item", sqlalchemy.String(64)], {}),
+                (["property", sqlalchemy.String(64)], {})
+            ]
+            chunk_size = 1
+
+            def requires(self):
+                return ModBaseTask()
+
+        class UpdateSQLATask(sqla.CopyToTable):
+            connection_string = CONNECTION_STRING
+            table = "item_property"
+            reflect = True
+            chunk_size = 1
+
+            def requires(self):
+                return ModSQLATask()
+
+            def copy(self, conn, ins_rows, table_bound):
+                ins = table_bound.update().\
+                    where(table_bound.c.property == sqlalchemy.bindparam("_property")).\
+                    values({{table_bound.c.item: sqlalchemy.bindparam("_item")}})
+                conn.execute(ins, ins_rows)
+
+            def rows(self):
+                for task in TASK_LIST:
+                    yield task.strip("\n").split("\t")
+
+        # Running only task1, and task2 should fail
+        task1, task2, task3 = ModBaseTask(), ModSQLATask(), UpdateSQLATask()
+        luigi.build([task1, task2, task3], local_scheduler=True, n_workers=self.NUM_WORKERS)
         self._check_entries(self.engine)
+
+
+class TestSQLA2(TestSQLA):
+    """ 2 workers version
+    """
+    NUM_WORKERS = 2

```

Comment:
You might want to add a test that runs many tasks too. But it's optional (sorry realize this PR has been going on for a long time already)